blockchain/stake/tickets_test.go

```diff
@@ -40,6 +40,49 @@ func calcHash256PRNGIVFromHeader(header *wire.BlockHeader) (chainhash.Hash, erro
 	return CalcHash256PRNGIV(hB), nil
 }
 
+// mockStakeParams implements the StakeParams interface and is used throughout
+// the tests to mock networks.
+type mockStakeParams chaincfg.Params
```

Comment:
wouldn't it be simpler to embed chaincfg.Params and then override if you need to?

Reply:
The idea was to make it completely separable from `chaincfg`, even in the tests.  I kept it defined in terms of `chaincfg` this go around to more quickly get the PR ready.  The plan was that in the next major version bump of `chaincfg` this will just be changed over to a struct with the relevant fields defined and it'll just work regardless of whatever happens to `chaincfg`.